Core/HLE/sceAtrac.cpp

```diff
@@ -1964,6 +1964,8 @@ static const u16 at3plusHeaderMap[][3] = {
 	{ 0x4524, 0x0, 0x0 },
 	{ 0x02E8, 0x1, 0x0 },
 	{ 0x5C24, 0x0, 0x0 },
+	{ 0x0118, 0x1, 0x0 },
+	{ 0x2224, 0x0, 0x0 },
```

Comment:
Well, no need to make a duplicate of the 0x2224 one for that. Just add 0x0118 next to 0x0180, good enough, the pairs don't matter.